samples/subsys/mgmt/hawkbit/src/main.c

```diff
@@ -36,6 +36,11 @@ int main(void)
 
 	ret = hawkbit_init();
 
+#ifdef CONFIG_HAWKBIT_SET_SETTINGS_RUNTIME
```

Comment:
wouldn't this get set by `check_hawkbit_server`?

Reply:
I don't really understand what you mean

Reply:
I mean isn't this change superfluous if it's setting it to the Kconfig value, since `check_hawkbit_server` is expected to use them as a fallback?

Reply:
yes it would be set by `check_hawkbit_server` and is redundant. But for a sample we want to show how to set the values, thats why its there.

Reply:
fair enough